### DISCUSSION TITLE
I need assistance to recover an account

### DISCUSSION THREAD

**Discussion by marioespnic** [community/community#137085] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Other

### Body

our company is using an account to manage some accounts, but, we lost the 2MFA and it was never used on CLI/Access Token. so we cannot recover it using the process provided online. I submitted a ticket 6 days ago, but, no response yet.

Can any staff give me a hand?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @marioespnic, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note for future discussions: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
